sample/src/commonMain/kotlin/App.kt

```diff
@@ -31,6 +32,7 @@ fun App() {
         }
         Column(
             modifier = Modifier
+                .safeDrawingPadding()
```

Comment:
You should add the `safeDrawingPadding()` to the `Text` modifier instead of the Column containing the text, so the text can be drawn behind the system bars (which is the main point of going edge-to-edge).